### PR TITLE
create roles and change year in footer

### PR DESCRIPTION
**Pull request by SetterVet**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ class User < ApplicationRecord
   belongs_to :project
   has_many :events, dependent: :destroy
   scope :users_joins_project, -> { includes(:project) }
+  enum role: %i[user admin]
```
[the remark this answers is not in the document]
Comment (SetterVet) [reply app/models/user.rb]: @afurm simple_user.role = ?

**Comment by rubycop** [comment]
Also add tests

**Comment by SetterVet** [comment]
In facts, all works correctly but not ideally( i will fix can-edit-event-params in views) and start write tests @VasylYaremkiv @rubycop @afurm @sonishc

**Comment by SetterVet** [comment]
all policies working correct, but not ideal. I rewrite all failed tests , adds them contexts . for simple-user and admin, now all tests passed

**Comment by SetterVet** [comment]
ready to review @rubycop @afurm @VasylYaremkiv @sonishc

File: app/javascript/components/users/UserEvents.vue [hunk 2]
```diff
@@ -60,7 +60,8 @@
       }
     },
     props: {
-      eventsForProps: { type: String }
+      eventsForProps: { type: String },
+      canEditEventParams: { type: Boolean }
```
Comment (VasylYaremkiv) [inline app/javascript/components/users/UserEvents.vue]: rename to `canEditEvent`

File: app/policies/application_policy.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  class Scope
```
Comment (VasylYaremkiv) [inline app/policies/application_policy.rb]: Do we use this `Scope` class? If no, please remove

**Comment by SetterVet** [comment]
fixed
 @rubycop @afurm @VasylYaremkiv @sonishc

**Comment by SetterVet** [comment]
squashed @VasylYaremkiv

**Pull request merged**